packages/contracts-bedrock/test/supersim/readme.md

```diff
@@ -0,0 +1,39 @@
+# Supersim End-to-End Test
+
+This guide describes how to run a manual L2-to-L2 message relay test using the `GasTank` and `MessageSender` contracts. Inspired by [this](https://supersim.pages.dev/guides/interop/cast#cast-commands-to-relay-interop-messages) guide.
+
+## Prerequisites
+
+- `foundry` and `go` must be installed.
+- A local `supersim` instance must be running. From the `optimism` repository root, start it with the following command. This ensures `supersim` uses the correct compiled contract artifacts.
+  ```bash
+  supersim --interop.l2tol2cdm.override "./packages/contracts-bedrock/forge-artifacts/L2ToL2CrossDomainMessenger.sol/L2ToL2CrossDomainMessenger.json"
```

Comment:
@hexshire if you install latest version of supersim it should include the updated l2tol2cdm, so no need to pass the `--interop.l2tol2cdm.override` flag.